Source/WebCore/css/CSSProperties.json

```diff
@@ -452,6 +452,19 @@
                 "url": "https://www.w3.org/TR/css-display-3/#the-display-properties"
             }
         },
+        "field-sizing": {
+            "codegen-properties": {
+                "parser-grammar": "fixed | content",
+                "settings-flag": "cssFieldSizingEnabled"
+            },
+            "specification": {
+                "category": "css-ui",
+                "url": "https://drafts.csswg.org/css-ui/#field-sizing"
+            },
+            "status": {
+                "status": "in development"
+            }
```

Comment:
```suggestion
            "status": "in development"
```
This also works

Reply:
Ah that's good to know. Should I change it or is it okay as is?

Reply:
I would prefer using the shorter form